mlos_bench/mlos_bench/services/config_persistence.py

```diff
@@ -9,7 +9,6 @@
 """
 
 import os
-import re
```

Comment:
Thank you.  Missed that during PR cleanup in #313 and forgot to get back to it :)